tests/driver_dcf77/Makefile.ci

```diff
@@ -0,0 +1,3 @@
+BOARD_INSUFFICIENT_MEMORY := \
+    stm32f030f4-demo \
```

Comment:
stm32f030f4-demo is the only board with insufficient memory?

Reply:
Murdock does not complain and `stm32f030f4-demo` really only has 16k ROM.